doc/user_guides/first_app_forward_voicemail.md

```diff
@@ -0,0 +1,303 @@
+# A First App: Let’s Forward Voicemail
+
+One of the things that we at 2600Hz pride ourselves on is the fact that KAZOO's open core means everyone, including us, uses the same APIs to develop applications. One of our longtime engineers, James Aimonetti, had this to say once about building on KAZOO versus other platforms:
+
+> KAZOO offers the [open core](https://en.wikipedia.org/wiki/Open_core) model, so the majority of typical APIs are built into the open source project. Some of the MonsterUI applications are closed [source], true, but the APIs they build on are open for you to build on as well. Nothing in KAZOO checks whether it’s MonsterUI making the API call vs another client. So anything you might see demoed [by 2600Hz] is possible.
+
+With that in mind, let’s walk through an example application that allows us to view, save to local disk, or delete voicemails from an account, with just the click of a couple buttons. We’re going to keep the code as simple as possible, and then get into more complex tasks in the future.
+
+
+NOTE: Be warned that deleting voicemails is a permanent action. Only delete test voicemails! Saving VMs is non-destructive, but we still recommend only using test voicemails for this guide.
+
+
+## Prerequisites
+
+---
+
+This guide assumes that you have:
+
+1. A KAZOO account
+2. A created user
+3. A registered device that can make outbound calls
+4. An auth token (see [API Basics](../../applications/crossbar/basics.md) for how to get one)
+
+If you need help with any of the prerequisites, you can check out our [Community](https://forums.2600hz.com/forums) for guides and discussions about troubleshooting issues.
+
+## Skeleton Code
+
+Provided is an HTML file to render the SMS that we receive from the KAZOO API, and also giving us clickable buttons that allow the user to forward a voicemail to the inputted address or delete the voicemail. The file calls a script, `api.js`, that we will define in the below section.
+
+**index.html**
+
+```html
+<head>
```

Comment:
I assume it might be helpful to package the snippets in this doc into a complete HTML/JS/CSS/whatever set of files so folks can download them to modify at their leisure?

Reply:
Somewhere like `doc/user_guides/code_examples/` ?